.env

```diff
@@ -13,10 +13,11 @@ REACT_APP_PREVIEW_REACT_URL=https://preview.keenthemes.com/metronic8/react
 REACT_APP_PREVIEW_DOCS_URL=https://preview.keenthemes.com/metronic8/react/docs
 REACT_APP_THEME_API_URL=https://preview.keenthemes.com/theme-api/api
 
-REACT_APP_PUBLIC_URL=http://localhost:3011
```

Comment:
Avoid change this file

Reply:
.env file added to .gitignore file... be careful because .env file is in delete mode